pkgs/development/tools/misc/coreboot-toolchain/default.nix

```diff
@@ -0,0 +1,100 @@
+{ lib, stdenvNoCC, fetchurl, fetchgit,
+  gnumake, patch, zlib, git, bison,
+  flex, gnat11, curl, perl, binutils,
+  coreutils
+}:
+
+let
+  version_coreboot	= "4.14";
+
+  # from util/crossgcc/buildgcc
```

Comment:
I would prefer if we could have an update-script like [this one](https://git.petabyte.dev/petabyteboy/corenix/raw/branch/main/pkgs/coreboot/update.sh)

Reply:
@petabyteboy Is it okay if I adjust and use your script for this package?

Reply:
Sure!

Reply:
I would like to leave it at the for now and rework it in another step. I created a separate PR #137702.